collect_app/src/main/res/layout/activity_forms_download_error.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should this be a `ConstraintLayout`? I'm still not the biggest fan of them, but it does seem like we're meant to be using them everywhere.

Reply:
There is a problem with including our toolbar (which uses <merge>) in constraint layouts. I think it should be addressed in a separate issue what I will do. https://stackoverflow.com/questions/46447621/using-include-with-merge-in-constraintlayout